.prettierrc.json

```diff
@@ -0,0 +1,18 @@
+{
+  "arrowParens": "always",
+  "bracketSpacing": true,
+  "embeddedLanguageFormatting": "auto",
+  "endOfLine": "lf",
+  "htmlWhitespaceSensitivity": "css",
+  "insertPragma": false,
+  "jsxSingleQuote": false,
+  "proseWrap": "preserve",
+  "quoteProps": "as-needed",
+  "requirePragma": false,
+  "singleQuote": true,
+  "tabWidth": 2,
```

Comment:
Shouldn't we use `4` instead?

Reply:
2 is my own preference, 4 can be a bit more space for clarity.

Reply:
> 2 is my own preference, 4 can be a bit more space for clarity.

Let's see what the other maintainers think. 🙂

Reply:
Let’s go with 4